src/comment-upserter.ts

```diff
@@ -2,15 +2,13 @@ import * as core from '@actions/core'
 import {RestEndpointMethods} from '@octokit/plugin-rest-endpoint-methods/dist-types/generated/method-types.d'
 import markdownEscape from 'markdown-escape'
 
-import {MentionRule} from './configuration'
+import {CommentConfiguration, MentionRule} from './configuration'
 import {Repo} from './github-types'
 
-export const HEADER = [
-  '<!-- codemention header -->',
-  '[CodeMention](https://github.com/tobyhs/codemention):\n',
-  '| File Patterns | Mentions |',
-  '| - | - |\n'
-].join('\n')
+export const HEADER = '<!-- codemention header -->'
```

Comment:
This is probably an extremely nitpicky thing, but I believe this comment header will no longer have a newline character separating it from the content afterwards. It probably doesn't make a difference when the content is rendered, but I do prefer a newline afterwards.

Reply:
Nice catch! Removing this is a hypothesis about what is responsible for the extra newline behavior we were seeing in the iOS notification:

![img_0757_720-2](https://github.com/user-attachments/assets/121c0e37-5d94-48e0-80d6-0c682239e20d)


It doesn't seem to affect the [comment on the web](https://github.com/expo/expo/pull/31869#issuecomment-2394106225), just the one in a notification.

Reply:
For reference, here's another github app notification for a normal review request: 
![Screenshot 2024-10-07 at 2 27 06 PM](https://github.com/user-attachments/assets/c67db09f-7bdd-4976-bb12-7763138ad01b)

(note that the newlines between "Why" and the rest of the content are expected and part of the summary)

Reply:
In that case, then I think it makes sense to not have the newline after the comment.

Reply:
Actually, after merging and testing this, it looks like the link doesn't render correctly on https://github.com/tobyhs/codemention-test/pull/8#issuecomment-2398797233

Reply:
Hmm. Yep, definitely something up. Feel free to revert and I'll test more tomorrow and find a fix!

Reply:
Fix: https://github.com/tobyhs/codemention/pull/16

(going to move the header into a footer to fix both issues in a follow-up)